paasta_tools/automatic_rollbacks/slack.py

```diff
@@ -259,8 +259,11 @@ def get_available_buttons(self) -> List[str]:
         return buttons
 
     def update_slack_thread(self, message, color=None):
-        print(f"Updating slack thread with: {message}")
-
+        if self.slack_client is None:
+            print(message)
```

Comment:
perhaps keep the print format the same, or be explicit that this isn't going to slack:

```
print(f"Would update slack thread with: {message}")
```